### PR TITLE
Add main console app

### PR DESCRIPTION
**Pull request by elassis**
# What I've Accomplished 🏁 
- Add the Main file to run the console app ✔️ 
- List all books.✔️
- List all people.✔️
- Create a person (teacher or student, not a plain Person).✔️
- Create a book.✔️
- Create a rental.✔️
- List all rentals for a given person id.✔️

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+    print 'Insert Name: '
+    name = gets.chomp
+    print 'Insert Age: '
+    age = gets.chomp
+    print 'Insert Specialization: '
+    spec = gets.chomp
+    @list_people.push(Teacher.new(age, name, spec))
+    puts 'Teacher created Succesfully'
+    show_menu
+  end
```
Comment (mikethreels) [inline main.rb]: - [ ] great job creating the UI 👍 However when creating a student or teacher the user received below error, please fix this before requesting another code review.

afbeelding

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Enmanuel, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.rb (hunk 1 above)
Comment (elassis) [reply main.rb]: thanks, resolve it!

**Review by mikethreels — APPROVED** [review]
Hi Enmanuel, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 :
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**